java/arcs/core/data/util/ReferencablePrimitive.kt

```diff
@@ -31,7 +31,8 @@ data class ReferencablePrimitive<T>(
     val valueRepr: String = value.toString()
 ) : Referencable {
     // TODO: consider other 'serialization' mechanisms.
-    override val id: ReferenceId = "Primitive<$klass>($valueRepr)"
+    override val id: ReferenceId =
+        "Primitive<${primitiveKClassMap.getOrElse(klass, klass::toString)}>($valueRepr)"
```

Comment:
If we add more primitive types in the future are we at risk of quietly/silently becoming non-performant again for those, with the `getOrElse`? Maybe we should fall back to "unknown" and just the class name (or something inexpensive)?

Reply:
klass::toString is the way to get kotlin class name at runtime but it's really slow due to method reflection.
Using either "unknown" or klass::toString would always fail on unknown classes or new primitives at run-time if 
we forgot to add new primitive types here because the unwrap() function would return null on unknown classes.

"unknown" cannot tell us what types we forgot to implement here from error log messages but klass::toString can.